src/SamplesApp/SamplesApp.UITests/TestFramework/ImageAssert.cs

```diff
@@ -235,4 +279,51 @@ public static void DoesNotHaveColorAt(FileInfo screenshot, float x, float y, Col
 		private static string ToArgbCode(Color color)
 			=> $"{color.A:X2}{color.R:X2}{color.G:X2}{color.B:X2}";
 	}
+
+	public struct ExpectedPixels
+	{
+		public static ExpectedPixels At(string name, float x, float y)
+			=> new ExpectedPixels(name, x, y, new Color[0, 0]);
+
+		public static ExpectedPixels At(float x, float y, [CallerLineNumber] int line = -1)
+			=> new ExpectedPixels($"at line: {line}", x, y, new Color[0,0]);
+
+		public ExpectedPixels Pixels(string[,] pixels) => new ExpectedPixels(Name, X, Y, pixels, Tolerance);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename the parameter 'pixels' so that it does not duplicate the method name.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4919453)